apps/endpoint-registry/src/services/registry/daos/postgress/Registry.ts

```diff
@@ -31,6 +32,23 @@ class Registry implements iRegistry {
   release(): void {
     this.client.release();
   }
+
+  async createAsset(asset: ICreateAsset): Promise<ICreateAsset> {
```

Comment:
Note: createAsset is also being executed from registerAas().

Actually the registerAas() performs composite actions (insertAsset, insertAas and instertEndpoint) and we should separate them (in following revision)